### ISSUE TITLE
I get trouble when I install my rockpi4b with the 21.12 image, it can't boot from emmc but can SD card.

### ISSUE THREAD

**Issue by 332518853** [manjaro-arm/rockpi4b-images#4]
When I updated my rockpi4b to version 21.12 of the image, my rockpi4b  would not boot from the emmc, but it would boot when I flashed it to the sd card

**Comment by Strit** [comment]
Yes. The 21.12 does not boot from eMMC. We are aware of the issue and are troubleshooting to find a solution.

**Comment by 332518853** [comment]
> 
You have worked hard, thank you for your work
